Traceback (most recent call last):
  File "e:\Python\Program\errorExample.py", line 4, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
